### ISSUE TITLE
「Animal Crossing New Horizons 」The application is closed

### ISSUE THREAD

**Issue by hg668zzx** [Ryujinx/Ryujinx-Games-List#4298]
##### RAM:8GB

After starting the game, there was a brief black screen, and then the program was shut down.

```
HLE.GuestThread.16 Application : Unhandled exception caught: Ryujinx.HLE.Exceptions.GuestBrokeExecutionException: The guest program broke execution!
   at Ryujinx.HLE.HOS.Kernel.SupervisorCall.Syscall.Break(UInt64 reason) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.HLE/HOS/Kernel/SupervisorCall/Syscall.cs:line 1852
   at Ryujinx.HLE.HOS.Kernel.SupervisorCall.SyscallDispatch.Break64(Syscall syscall, IExecutionContext context) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.HLE/Ryujinx.Horizon.Generators/Ryujinx.Horizon.Generators.Kernel.SyscallGenerator/SyscallDispatch.g.cs:line 1806
   at Ryujinx.HLE.HOS.Kernel.SupervisorCall.SyscallDispatch.Dispatch64(Syscall syscall, IExecutionContext context, Int32 id) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.HLE/Ryujinx.Horizon.Generators/Ryujinx.Horizon.Generators.Kernel.SyscallGenerator/SyscallDispatch.g.cs:line 3568
   at Ryujinx.HLE.HOS.Kernel.SupervisorCall.SyscallHandler.SvcCall(IExecutionContext context, UInt64 address, Int32 id) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.HLE/HOS/Kernel/SupervisorCall/SyscallHandler.cs:line 17
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.SynchronousException(HvMemoryManager memoryManager, HvVcpu& vcpu) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 182
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.Execute(HvMemoryManager memoryManager, UInt64 address) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 160
   at Ryujinx.Cpu.AppleHv.HvCpuContext.Execute(IExecutionContext context, UInt64 address) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.Cpu/AppleHv/HvCpuContext.cs:line 30
   at Ryujinx.HLE.HOS.ArmProcessContext`1.Execute(IExecutionContext context, UInt64 codeAddress) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.HLE/HOS/ArmProcessContext.cs:line 40
   at Ryujinx.HLE.HOS.Kernel.Threading.KThread.ThreadStart() in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.HLE/HOS/Kernel/Threading/KThread.cs:line 1265
   at System.Threading.Thread.StartCallback()

```

#### Screenshots:

no

#### Log file:
[Ryujinx_ 1.1.0-macos1_2022-12-20_14-14-10.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/10265881/Ryujinx_.1.1.0-macos1_2022-12-20_14-14-10.log)

**Comment by AcK77** [comment]
Remove all cheats you have added and try again. I suggest you to come at our discord for support. If you want to really report the current state of the game, you can do it in the issue which already exists instead of opening a new one. Thanks.

**Comment by hg668zzx** [comment]
i delete all of cheats ,but still closed












Villan
***@***.***






---Original---
From: ***@***.***&gt;
Date: Tue, Dec 20, 2022 15:17 PM
To: ***@***.***&gt;;
Cc: "Villan ***@***.******@***.***&gt;;
Subject: Re: [Ryujinx/Ryujinx-Games-List] 「Animal Crossing New Horizons 」The application is closed (Issue #4298)




 
Remove all cheats you have added and try again. I suggest you to come at our discord for support. If you want to really report the current state of the game, you can do it in the issue which already exists instead of opening a new one. Thanks.

**Comment by hg668zzx** [comment]
i delete all cheats,but it still doesn't work.
